src/Strategy/DatabaseMigrate.php

```diff
@@ -81,7 +81,7 @@ public function migrate(Config $config): void
             if (in_array($schemaName, self::SKIP_CLONE_SCHEMAS, true)) {
                 continue;
             }
-            if (str_starts_with($schemaName, 'WORKSPACE')
+            if (preg_match('/^([0-9]*_)?WORKSPACE/', $schemaName)
```

Comment:
Dal bych `\d` místo `[0-9]` a asi `+` místo `*`, když před `_` čekáme číslo.

```suggestion
            if (preg_match('/^([\d+_)?WORKSPACE/', $schemaName)
```

Reply:
jo + určitě 👍